test/bootstrap/config.php

```diff
@@ -0,0 +1,70 @@
+<?php
+
+return array (
+  'all' =>
+  array (
+    'propel' =>
+    array (
+      'class' => 'sfPropelDatabase',
+      'param' =>
+      array (
+        'encoding' => 'utf8mb4',
+        'persistent' => true,
+        'pooling' => true,
+        'dsn' => 'mysql:host=127.0.0.1;port=63003;dbname=atom;charset=utf8mb4',
+        'username' => 'atom',
+        'password' => 'atom_12345',
```

Comment:
Is having the password in plain text a security risk?

Reply:
I don't think it is ...

It's just an empty database for the tests and we already have that password in the code as default for the development env. Also the VM is up only for around a few minutes and guess it's not accessible from the outside and even if it's, the Percona port is only accessible internally:

https://github.com/artefactual/atom/blob/qa/2.x/docker/docker-compose.dev.yml#L57